cdc_vaccines/delphi_cdc_vaccines/run.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+"""Functions to call when running the function.
+
+This module should contain a function called `run_module`, that is executed
+when the module is run with `python -m MODULE_NAME`.  `run_module`'s lone argument should be a
+nested dictionary of parameters loaded from the params.json file.  We expect the `params` to have
+the following structure:
+    - "common":
+        - "export_dir": str, directory to which the results are exported
+        - "log_filename": (optional) str, path to log file
+    - "indicator": (optional)
+        - "wip_signal": (optional) Any[str, bool], list of signals that are works in progress, or
+            True if all signals in the registry are works in progress, or False if only
+            unpublished signals are.  See `delphi_utils.add_prefix()`
+        - Any other indicator-specific settings
+"""
+from datetime import timedelta, datetime
+from itertools import product
+import time as tm
+import os
+import numpy as np
+from delphi_utils.export import create_export_csv
+from delphi_utils.geomap import GeoMapper
+from delphi_utils import get_structured_logger
+from .constants import GEOS, SIGNALS, SMOOTHERS
+from .pull import pull_cdcvacc_data
+
+
+def run_module(params):
+    """
+    Run the indicator.
+
+    Arguments
+    --------
+    params:  Dict[str, Any]
+        Nested dictionary of parameters.
+    """
+    start_time = tm.time()
+    logger = get_structured_logger(
+        __name__, filename=params["common"].get("log_filename"),
+        log_exceptions=params["common"].get("log_exceptions", True))
+    base_url = params["indicator"]["base_url"]
+    ## build the base version of the signal at the most detailed geo level you can get.
+    all_data = pull_cdcvacc_data(base_url, logger)
+    run_stats = []
+    ## aggregate & smooth
+
+
+    if not os.path.exists(params["common"]["export_dir"]):
+        os.makedirs(params["common"]["export_dir"])
+
+    for (sensor, smoother, geo) in product(SIGNALS, SMOOTHERS, GEOS):
+
+        logger.info("Running on ",
+            sensor=sensor,
+            smoother=smoother,
+            geo=geo)
+        geo_map = geo
+        if geo=='state':
+            geo_map='state_code'
+
+        df = GeoMapper().replace_geocode(
+            all_data[['timestamp','fips', sensor]],
+            from_col='fips',
+            from_code="fips",
+            new_col="geo_id",
+            new_code=geo_map,
+            date_col="timestamp")
+        df["val"] = df[["geo_id", sensor]].groupby("geo_id")[sensor].transform(
+            smoother[0].smooth
+        )
+        df["se"] = np.nan
+        df["sample_size"] = np.nan
+        sensor_name = sensor + smoother[1]
+        print(sensor_name)
```

Comment:
Also, if you anticipate any nans in the dataset that will be ingested into the db, consider annotating them with nancodes in the missing columns, like [here](https://github.com/dshemetov/covidcast-indicators/blob/263485b8d4d94a36ebd9ed6292d29b62b69efc64/jhu/delphi_jhu/run.py#L67).

Reply:
Should I remove these columns? I kept them because they were in the USAFacts indicator, but I can remove them if needed.

Reply:
Oh no, I mean that we now support three extra columns `missing_value`, `missing_stderr`, and `missing_sample_size`, where we write nancodes for why a value is missing. See the link for an example. The possible nancodes are [here](https://github.com/dshemetov/covidcast-indicators/blob/263485b8d4d94a36ebd9ed6292d29b62b69efc64/_delphi_utils_python/delphi_utils/nancodes.py). So for example, this indicator is not a sample-based indicator so stderr and sample_size just do not apply, so we would write `Nans.NOT_APPLICABLE` in the corresponding columns.

Reply:
Since this is a new indicator, we should definitely be creating it with the correct Nans codes in the missing_* columns as needed. It sounds like this indicator should use:
* `missing_stderr = Nans.NOT_APPLICABLE`
* `missing_sample_size = Nans.NOT_APPLICABLE`